packages/rspack/src/compiler.ts

```diff
@@ -669,6 +669,16 @@ class Compiler {
 				resolveData
 			);
 
+		NormalModule.getCompilationHooks(this.compilation).loader.tap(
```

Comment:
I don't find an easy way to implement https://github.com/webpack/webpack/blob/5b1fd3ebbaf0fd5b7706c29f0be12b7b0150e743/lib/NormalModule.js#L672-L672